core/src/main/java/com/microsoft/applicationinsights/internal/util/DeviceInfo.java

```diff
@@ -65,13 +63,15 @@ public static String getLocale()
         final String languageTagMethodName = "toLanguageTag";
         Locale defaultLocale = Locale.getDefault();
         try {
+            // FIXME toLanguageTag is since 1.7; reflection isn't needed anymore.
```

Comment:
lets remove reflection if there is lang support already. I have no objections if you make this change as part of this PR.